operator/main.go

```diff
@@ -46,10 +46,26 @@ func init() {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
-	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
+	var provisioningPodImage string
+	flag.StringVar(
+		&metricsAddr,
+		"metrics-addr",
+		":8080",
+		"The address the metric endpoint binds to.",
+	)
+	flag.BoolVar(
+		&enableLeaderElection,
+		"enable-leader-election",
+		false,
 		"Enable leader election for controller manager. "+
-			"Enabling this will ensure there is only one active controller manager.")
+			"Enabling this will ensure there is only one active controller manager.",
+	)
+	flag.StringVar(
+		&provisioningPodImage,
+		"provisioning-pod-image",
+		"metabroker-provisioning",
```

Comment:
We'll probably want to tweak this before this moves out of prototyping (because of Docker Hub quotas).

Reply:
No, this is not even a valid Docker Hub image repository. I'll leave a `TODO` just in case, so I don't forget to set this to empty and perform a validation (with defaults set to the correct image in the future chart that is to come).

Reply:
Added TODO in d6bc946.